tests/Makefile

```diff
@@ -76,6 +76,7 @@ endif
 ifdef MYST_NIGHTLY_TEST
 DIRS += ltp
 DIRS += sockperf
+DIRS += ltp2
```

Comment:
Do you plan to consolidate ltp2 and ltp into one test?

Reply:
Yes, once I separate out the fstests I will be doing that as well